Cakes-Co/css/style.css

```diff
@@ -0,0 +1,312 @@
+/*  Add your styling here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --pink: rgb(242 200 217);
+  --darker-pink: rgb(201, 40, 105);
+  --light-pink: rgb(249 237 244);
+  --background-color: var(--white);
+  --gold: rgb(160 150 111);
+  --white: rgb(255, 255, 255);
+
+  --page-width: 100vw;
+
+  --default-font-family: "Nunito Sans", sans-serif;
+  --heading-font-family: "Playfair Display", serif;
+
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--darker-pink);
+  font-family: var(--default-font-family);
+}
+
+.container {
+  margin: 0 auto;
+  position: relative;
+}
+
+a {
+  text-decoration: none;
+  color: var(--gold);
+  transition: all 0.2s ease-in-out;
+}
+
+li {
+  list-style: none;
+}
+
+h1 {
+  font-family: var(--heading-font-family);
+  margin: 0 0 0.5rem 0;
+}
+
+h3 {
+  font-weight: 100;
+  font-style: italic;
+  align-self: flex-end;
+}
+
+/* Header */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  gap: 1rem;
+  align-items: center;
+  background-color: var(--light-pink);
+  padding: 1rem;
+}
+
+.logo {
+  max-width: 100px;
+}
+
+.branding {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.branding p {
+  font-family: var(--heading-font-family);
+  font-size: 1.4rem;
+  /* margin-top: -20px; */
+  /* text-shadow: 0 -1px var(--gold); */
```

Comment:
Best to remove code you no longer need not comment it out, see the CYF style guide

Reply:
✓ removed them